pkgs/servers/nextcloud/packages/nextcloud-apps.json

```diff
@@ -8,6 +8,7 @@
 , "end_to_end_encryption": "agpl3Plus"
 , "files_texteditor": "agpl3Plus"
 , "files_markdown": "agpl3Plus"
+, "files_mindmap": "agpl3Only"
```

Comment:
You're sure about the license?
https://github.com/ACTom/files_mindmap/blob/411db10d6f4bd325c56bd01dafca31fef4172d91/lib/Controller/PublicFileHandlingController.php#L7

Reply:
I opened an upstream issue about this https://github.com/ACTom/files_mindmap/issues/205

And changed it to AGPL3Plus for now